resources/unicode-ranges.json

```diff
@@ -0,0 +1 @@
:[122880,122927]},{"category":"Nyiakeng Puachue Hmong","hexrange":["1E100","1E14F"],"range":[123136,123215]},{"category":"Wancho","hexrange":["1E2C0","1E2FF"],"range":[123584,123647]},{"category":"Mende Kikakui","hexrange":["1E800","1E8DF"],"range":[124928,125151]},{"category":"Adlam","hexrange":["1E900","1E95F"],"range":[125184,125279]},{"category":"Indic Siyaq Numbers","hexrange":["1EC70","1ECBF"],"range":[126064,126143]},{"category":"Ottoman Siyaq Numbers","hexrange":["1ED00","1ED4F"],"range":[126208,126287]},{"category":"Arabic Mathematical Alphabetic Symbols","hexrange":["1EE00","1EEFF"],"range":[126464,126719]},{"category":"Mahjong Tiles","hexrange":["1F000","1F02F"],"range":[126976,127023]},{"category":"Domino Tiles","hexrange":["1F030","1F09F"],"range":[127024,127135]},{"category":"Playing Cards","hexrange":["1F0A0","1F0FF"],"range":[127136,127231]},{"category":"Enclosed Alphanumeric Supplement","hexrange":["1F100","1F1FF"],"range":[127232,127487]},{"category":"Enclosed Ideographic Supplement","hexrange":["1F200","1F2FF"],"range":[127488,127743]},{"category":"Miscellaneous Symbols and Pictographs","hexrange":["1F300","1F5FF"],"range":[127744,128511]},{"category":"Emoticons","hexrange":["1F600","1F64F"],"range":[128512,128591]},{"category":"Ornamental Dingbats","hexrange":["1F650","1F67F"],"range":[128592,128639]},{"category":"Transport and Map Symbols","hexrange":["1F680","1F6FF"],"range":[128640,128767]},{"category":"Alchemical Symbols","hexrange":["1F700","1F77F"],"range":[128768,128895]},{"category":"Geometric Shapes Extended","hexrange":["1F780","1F7FF"],"range":[128896,129023]},{"category":"Supplemental Arrows-C","hexrange":["1F800","1F8FF"],"range":[129024,129279]},{"category":"Supplemental Symbols and Pictographs","hexrange":["1F900","1F9FF"],"range":[129280,129535]},{"category":"Chess Symbols","hexrange":["1FA00","1FA6F"],"range":[129536,129647]},{"category":"Symbols and Pictographs Extended-A","hexrange":["1FA70","1FAFF"],"range":[129648,129791]},{"category":"CJK Unified Ideographs Extension B","hexrange":["20000","2A6DF"],"range":[131072,173791]},{"category":"CJK Unified Ideographs Extension C","hexrange":["2A700","2B73F"],"range":[173824,177983]},{"category":"CJK Unified Ideographs Extension D","hexrange":["2B740","2B81F"],"range":[177984,178207]},{"category":"CJK Unified Ideographs Extension E","hexrange":["2B820","2CEAF"],"range":[178208,183983]},{"category":"CJK Unified Ideographs Extension F","hexrange":["2CEB0","2EBEF"],"range":[183984,191471]},{"category":"CJK Compatibility Ideographs Supplement","hexrange":["2F800","2FA1F"],"range":[194560,195103]},{"category":"Tags","hexrange":["E0000","E007F"],"range":[917504,917631]},{"category":"Variation Selectors Supplement","hexrange":["E0100","E01EF"],"range":[917760,917999]},{"category":"Supplementary Private Use Area-A","hexrange":["F0000","FFFFF"],"range":[983040,1048575]},{"category":"Supplementary Private Use Area-B","hexrange":["100000","10FFFF"],"range":[1048576,1114111]}]
```

Comment:
I wonder if we can format this file to make the diffing easier and minify it as a build step?!

Reply:
I guess we could, but also this doesn't get updated too often and would have to add some package like prettier as a dependency.